sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -420,7 +420,7 @@ public virtual Task<string[]> GetPartitionIdsAsync(CancellationToken cancellatio
                     return;
                 }
                 catch (EventHubsException eventHubException) when (eventHubException.Reason == EventHubsException.FailureReason.ClientClosed
-                                                                    && ShouldRecreateProducer(pooledProducer.TransportProducer, options.PartitionId))
+                                                                   && ShouldRecreateProducer(pooledProducer.TransportProducer, options.PartitionId))
```

Comment:
nit:  I'd recommend moving the `when`  clause to the next line and indenting.   That would keep the clause together and, I think, help readability.

Reply:
I put all the conditions on the same line, using the space now available.

Reply:
I find it more readable in the new way, yeah.   :)